app/controllers/application_controller.rb

```diff
@@ -1,3 +1,15 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def after_sign_in_path_for(current_user)
+    user_path(current_user)
+  end
+
+  def after_sign_up_path_for(current_user)
+    user_path(current_user)
+  end
+
+  def after_sign_out_path_for(current_user)
+   	users_path
```

Comment:
Indentation issue

Reply:
There is some problem with git! it sometimes shows different indentations from the original file in my editor. I'm looking into it.